addons/l10n_cn/views/report_voucher.xml

```diff
@@ -22,7 +22,7 @@
 
                     <div align="center">
                         <h2>
-                            <span>记账凭证</span>
+                            <span>Accounting Voucher</span>
```

Comment:
Not sure about the changes in this file: if someone updates l10n_cn , the translations for the module won't be reloaded, and this will suddenly break.  Modifying this is trickier than account's templates ; as most existing dbs won't need to create new companies, and hence won't use the accounting templates anymore. For me, this part of the diff must go to master or 16.1.